scripts/library.sh

```diff
@@ -482,7 +482,13 @@ function run_go_tool() {
   if [[ -z "$(which ${tool})" ]]; then
     local action=get
     [[ $1 =~ ^[\./].* ]] && action=install
-    go ${action} $1
+    # Avoid running `go get` from root dir of the repository, as it can change go.sum and go.mod files.
+    # See discussions in https://github.com/golang/go/issues/27643.
+    if [[ ${action} == "get" && $(pwd) == "${REPO_ROOT_DIR}" ]]; then
+      cd .. && go ${action} $1 && cd - || return 1
```

Comment:
this is brittle, can we do this?
```suggestion
      local temp_dir="$(mktemp -d)"
      local install_failed=0
      pushd "$(temp_dir)"
      go ${action} $1 || install_failed=1
      popd
      return ${install_failed}
```

Reply:
Done. Thanks.